README.md

```diff
@@ -48,63 +57,125 @@ python scripts/make.py "qt/bin" --lib-path "nomacs/3rd-party/build"
+
+# qt5
+# note: cmake configuration will fail if libquazip1-qt6-dev is also installed; or you can use USE_SYSTEM_QUAZIP=OFF
+sudo apt install qtbase5-dev qttools5-dev libqt5svg5-dev qt5-image-formats-plugins libexiv2-dev libraw-dev libopencv-dev libtiff-dev libtiff-dev libquazip1-qt5-dev build-essential git cmake lcov libgtest-dev kimageformat-plugins
+```
 
-Clone the nomacs repository from GitHub:
+#### Ubuntu 22.04
 
 ```console
-git clone https://github.com/nomacs/nomacs.git
+# qt6
+sudo apt install qt6-base-dev qt6-tools-dev qt6-tools-dev-tools libqt6svg6-dev libqt6core5compat6-dev qt6-l10n-tools qt6-image-formats-plugins libexiv2-dev libraw-dev libopencv-dev libtiff-dev libtiff-dev build-essential git cmake lcov libgtest-dev libgl-dev
+
+# qt5
+sudo apt install qtbase5-dev qttools5-dev libqt5svg5-dev qt5-image-formats-plugins libexiv2-dev libraw-dev libopencv-dev libtiff-dev libtiff-dev libquazip5-dev build-essential git cmake lcov libgtest-dev kimageformat-plugins
+```
+
+#### Arch
+
+```console
+# qt6
+sudo pacman --sync qt6-base qt6-imageformats qt6-svg qt6-tools quazip-qt6 kimageformats exiv2 libraw libtiff opencv kimageformats git cmake gtest base-devel make
+```
+
+#### Redhat/Fedora/CentOS (tested on Rocky 9.5)
+```console
+# qt6
+sudo dnf install qt6-qtbase-devel qt6-qtimageformats qt6-qtsvg-devel qt6-qttools-devel qt6-qt5compat-devel LibRaw-devel opencv-devel exiv2-devel libtiff-devel git cmake lcov gtest-devel gcc-c++
+
+# qt5
+sudo dnf install qt5-qtbase-devel qt5-qtimageformats qt5-qtsvg-devel qt5-qttools-devel LibRaw-devel opencv-devel exiv2-devel libtiff-devel git cmake lcov gtest-devel gcc-c++ quazip-qt5-devel
+```
+
+#### FreeBSD (14.2 release)
+
+```
+# qt6
+sudo pkg install qt6-base qt6-imageformats qt6-svg qt6-5compat qt6-tools quazip-qt6 tiff exiv2 kf6-kimageformats libraw opencv git cmake googletest gcc
+
+#qt5
+sudo pkg install qt5-gui qt5-imageformats
+qt5-svg qt5-linguisttools qt5-qmake qt5-buildtools qt5-uitools qt5-concurrent quazip-qt5 tiff exiv2 kf5-kimageformats libraw opencv git cmake googletest gcc
+```
+
+#### Haiku (r1 beta 5)
+```console
+# qt6
+pkgman install qt6_base_devel qt6_tools_devel qt6_svg_devel qt6_5compat_devel quazip1_qt6_devel tiff_devel libraw_devel opencv_devel gtest_devel exiv2_devel kimageformats6 qt6_imageformats cake git gcc make pkgconfig lcms_devel
 ```
 
-This will by default place the source into ~/nomacs
-Go to the nomacs/ImageLounge directory and run `cmake` to get the Makefiles:
+### Configure nomacs
+
+Nomacs is configured with cmake. These cmake options are often needed:
+- QT_VERSION_MAJOR=[5|6] - Default 5, 6 recommended
+- ENABLE_QUAZIP=[ON|OFF] - Default OFF, ON recommended
+- USE_SYSTEM_QUAZIP=[ON|OFF] - Default ON, recommended
+- CMAKE_BUILD_TYPE=[Debug|Release|RelWithDebInfo] - Debug builds have more logging as well as debug symbols.
```

Comment:
I suggest removing the recommendations here because quazip is an optional dependency.

```suggestion
- ENABLE_QUAZIP=[ON|OFF] - Default OFF
- USE_SYSTEM_QUAZIP=[ON|OFF] - Default ON
- CMAKE_BUILD_TYPE=[Debug|Release|RelWithDebInfo] - For normal usage, choose `RelWithDebInfo`. For development, `Debug` builds have more logging as well as debug symbols.
```

Reply:
I don't feel RellWithDebInfo is worthwhile, as it will build debug symbols.. but I guess they could appear in crash reports. Is that your thinking?

Reply:
Yes, stack traces will be available if the build has debug symbols.